### ISSUE TITLE
Plugin crashes on QGIS 3.03, OS X 10.13.5: "TypeError: QgsCoordinateTransform(): arguments did not match any overloaded call"

### ISSUE THREAD

**Issue by paul0207** [PANOimagen/profiletool#4]
TypeError: QgsCoordinateTransform(): arguments did not match any overloaded call:   overload 1: too many arguments   overload 2: not enough arguments   overload 3: not enough arguments   overload 4: not enough arguments   overload 5: argument 1 has unexpected type 'QgsCoordinateReferenceSystem' 
Traceback (most recent call last):
  File "/Users/paul/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/profiletool/ui/ptdockwidget.py", line 352, in refreshPlot
    self.profiletoolcore.pointstoDraw, False, True)
  File "/Users/paul/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/profiletool/tools/profiletool_core.py", line 174, in updateProfil
    self.profiles[i], _, _ = DataReaderTool().dataVectorReaderTool(self.iface, self.toolrenderer.tool, self.profiles[i], self.pointstoDraw, float(self.dockwidget.mdl.item(i,4).data(QtCore.Qt.EditRole)) )
  File "/Users/paul/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/profiletool/tools/dataReaderTool.py", line 211, in dataVectorReaderTool
    xform = QgsCoordinateTransform(sourceCrs, destCrs)
TypeError: QgsCoordinateTransform(): arguments did not match any overloaded call:
  overload 1: too many arguments
  overload 2: not enough arguments
  overload 3: not enough arguments
  overload 4: not enough arguments
  overload 5: argument 1 has unexpected type 'QgsCoordinateReferenceSystem'

Python version: 3.6.6rc1 (v3.6.6rc1:1015e38be4, Jun 12 2018, 01:14:33) [GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] 
QGIS version: 3.0.3-Girona Girona, exported

**Comment by JaviBecerra** [comment]
Hi @paul0207,
Thanks for reporting!
It seems that the call to QgsCoordinateTransform(sourceCrs, destCrs) was deprecated in 3.0, have to use QgsCoordinateTransform(sourceCrs, destCrs, QgsProject.instance()) instead (https://qgis.org/api/classQgsCoordinateTransform.html). We will update and publish a new version of the plugin fixing this issue asap.

**Comment by paul0207** [comment]
Javier, thanks for the quick response!

**Comment by JaviBecerra** [comment]
Hi @paul0207,
Roberto made a commit a couple days ago to address this issue, could you verify that it works for you before we push a new version to QGis repo?

**Comment by paul0207** [comment]
Javier,

I think I have found out the problem, I was selecting a Shapefile layer and this plugin is intended for raster layers, right?

Anyway I tried with a raster layer and it works.
But when clicking About I get:

ModuleNotFoundError: No module named 'ui.dlgabout' 
Traceback (most recent call last):
  File "/Users/paul/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/profiletool-master/profileplugin.py", line 113, in about
    from ui.dlgabout import DlgAbout
  File "/Applications/QGIS3.app/Contents/MacOS/../Resources/python/qgis/utils.py", line 664, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ModuleNotFoundError: No module named 'ui.dlgabout'

Thank you!
